tests/Handler/UploadHandlerTest.php

```diff
@@ -183,8 +183,7 @@ public function testRemove(): void
         $this->mapping
             ->expects($this->once())
             ->method('erase')
-            ->with($this->object)
-            ->willReturn(null);
+            ->with($this->object);
```

Comment:
Why we don't check anymore the null returned?

Reply:
Because of this:

```
7) Vich\UploaderBundle\Tests\Handler\UploadHandlerTest::testRemove
Method erase may not return value of type NULL, its return declaration is ": void"
```